pkg/validate/validate.go

```diff
@@ -166,3 +166,14 @@ func Files(fileNames []string, validationRules []*validationrule.ValidationRule,
 	validationReport.Duration = time.Since(start)
 	return validationReport
 }
+
+func generateExcludedRules(excludedRulesText string) []string {
+	excludedRules := strings.Split(excludedRulesText, ",")
+	for i := range excludedRules {
+		excludedRules[i] = strings.TrimSpace(excludedRules[i])
+	}
+	excludedRules = slices.DeleteFunc(excludedRules, func(s string) bool {
```

Comment:
I'd completely drop the `slices.DeleteFunc` if you agree and do this
```go
excludedRules := []string{}
for _, r := strings.Split(excludedRulesText, ",") {
    rule := strings.TrimSpace(r)
    if rule == "" {
        continue
    }
    excludedRules = append(excludedRules, rule)
}
```

To explain myself, I don't like the fact that `  foo,,    foo bar,   foo` would result into `foo,foobar,foo`. The space in the middle is not a whitespace noise but might be a regular rule name with a space if that would be acceptable, for example.

Reply:
I get it, and that's not the behavior, as you can see with this test case https://github.com/FUSAKLA/promruval/pull/89/files#diff-9746d4a5b310fcfb5b0e9bf8802c8d7bd7390bae0b5ef6b158efb6c64ed4c8b8R36 but I will implement the suggestion :)

Reply:
Sorry, my bed, I miss read the code. I somehow thought you were using the slices. Delete to iterate over string characters, not the whole rule names :facepalm:  

thanks anyway